.bluemix/deploy.json

```diff
@@ -6,30 +6,53 @@
   "type": "object",
   "properties": {
     "selected-region": {
-      "description": "The Bluemix region",
+      "description": "The IBM Cloud region",
       "type": "string"
     },
     "selected-organization": {
-      "description": "The Bluemix org",
+      "description": "The IBM Cloud org",
       "type": "string"
     },
-   "selected-space": {
-      "description": "The Bluemix space",
+    "selected-space": {
+      "description": "The IBM Cloud space",
       "type": "string"
     },
     "erp-app-name": {
+      "title": "ERP App Name",
       "description": "Name of the ERP API",
       "type": "string"
     },
-   "controller-app-name": {
+    "erp-app-branch": {
+      "title": "ERP App Branch",
+      "description": "GitHub branch of the ERP API",
+      "type": "string"
+    },
+    "controller-app-name": {
+      "title": "Controller App Name",
       "description": "Name of the controller app",
       "type": "string"
     },
+    "controller-app-branch": {
```

Comment:
given we are providing an option to set the branch name in the wizard, I don't get why we keep the "dev" pipeline in the toolchain

If I need a dev pipeline, I can simply run the toolchain again and change "master" to "dev" - this will make it consistent with Cloud Co. And in most cases, it would be enough to have just one toolchain